release_notes/ocp-4-11-release-notes.adoc

```diff
@@ -150,6 +150,44 @@ For more information, see link:https://kubernetes.io/docs/concepts/services-netw
 [id="ocp-4-11-storage"]
 === Storage
 
+[id="ocp-4-11-storage-csi-azure-file-ga"]
+==== Persistent storage using Microsoft Azure File CSI Driver Operator is generally available
+{product-title} is capable of provisioning persistent volumes (PVs) using the Container Storage Interface (CSI) driver for Azure File. This feature was previously introduced as a Technology Preview feature in {product-title} 4.10 and is now generally available and enabled by default in {product-title} 4.11.
+
+For more information, see ???
+
+[id="ocp-4-11-storage-csi-migration-cinder-ga"]
+==== Automatic CSI migration for OpenStack Cinder is generally available
+Starting with {product-title}  4.8, automatic migration for in-tree volume plug-ins to their equivalent Container Storage Interface (CSI) drivers became available as a Technology Preview feature. Support for Cinder was provided in this feature in {product-title} 4.8, and {product-title} 4.11 now supports automatic migration for Cinder as generally available.
```

Comment:
Should we mention that this is enabled by default? (no action required from the admin)

Reply:
Yes! I've added this.

Reply:
Thanks Lisa!